deployments/helm/k8s-dra-driver/templates/privilegedrolebinging.yaml

```diff
@@ -0,0 +1,17 @@
+# Apply only when running on OpenShift to let the kublet plugin run privileged
+{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "k8s-dra-driver.fullname" . }}-privileged-role-binding
```

Comment:
Does it make sense give this a name that has `openshift` in it, in case we need to add other RoleBindings in the future that aren't openshift specific.

Reply:
Makes sense.

Reply:
I'm not sure, actually, because it won't exists anywhere else except on OpenShift.